tasks/installcollections.js

```diff
@@ -0,0 +1,51 @@
+
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var Promise = require('es6-promise').Promise;
```

Comment:
Need to use `require('es6-promise').polyfill();` or `Promise` is undefined in denodeify.